src/flinkSql/flinkLanguageClientManager.ts

```diff
@@ -356,6 +355,18 @@ export class FlinkLanguageClientManager implements Disposable {
       return false;
     }
   }
+  /**
+   * Clear diagnostics for a specific document URI
+   * This is used to clear diagnostics when the document is closed or when the language client is reinitialized
```

Comment:
nit: or when the document has been changed

Reply:
Will augment when going into main, thanks.